docs/runtimes/README.md

```diff
@@ -15,10 +15,21 @@ This page is an introduction to the runtimes. The next sections (PHP functions,
 The name of the runtimes follow this pattern:
 
 ```
-arn:aws:lambda:<region>:416566615250:layer:<layer-name>:<layer-version>
+arn:aws:lambda:<region>:<account-id>:layer:<layer-name>:<layer-version>
 ```
 
-Bref offers the following runtimes (`<layer-name>`):
+### Supported regions (`<region>`)
+
+- `us-east-2`
+- other regions will be supported soon
+
+Remember to use the `<region>` that matches your application, else Lambda will not find the layer.
+
+### Account
+
+Bref Account id is: `416566615250`
```

Comment:
I think having a placeholder + section for that is not necessary, it is simpler without it. Could you re-insert the account ID in the pattern above?

Reply:
Done